styles.css

```diff
@@ -0,0 +1,37 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+	margin:0;
+	font-size:10px;
+	background-color:#000000;
+}
+nav {
+	width: 100%;
+	position: fixed;
+	top: 20px;
+}
+
+.centerNavBar {
+	background-color: #1c1c1c;
+	border-radius: 30px;
+	width: 800px;
+	margin: 0 auto;
+	overflow: auto;
+	padding-bottom: 10px;
+}
+		
+nav li {
+	font-family: 'Lobster', 'cursive';
+	 float:left;
+	 color: #ffffff;
+     list-style: none;
+	 margin-right: 90px;
+	 margin-left:20px;
+	 font-size: 1.5rem;
+ }
+ 
+nav li:hover {
+	color:#b69a46;
+} 
```

Comment:
Charlie would say it needs the media queries to make it responsive to different screen sizes, then it can be signed off :)